nullaway/src/main/java/com/uber/nullaway/handlers/RxNullabilityPropagator.java

```diff
@@ -76,10 +74,9 @@
           // (must take exactly 1 argument)
           .withFilterMethodFromSignature("filter(io.reactivex.functions.Predicate<? super T>)")
           // Names and relevant arguments of all the methods of io.reactivex.Observable that behave
-          // like
-          // .map(...) for the purposes of this checker (the listed arguments are those that take
-          // the
-          // potentially filtered objects from the stream)
+          // like .map(...) for the purposes of this checker (the listed arguments are those that
+          // take
+          // the potentially filtered objects from the stream)
```

Comment:
Nit: can we get rid of the gratuitous newline? 😄

Reply:
Sure! Maybe this is the first time this file ever got GJF-ed somehow?